pkg/pillar/cmd/volumemgr/handleresolve.go

```diff
@@ -13,7 +13,7 @@ func MaybeAddResolveConfig(ctx *volumemgrContext, cs types.ContentTreeStatus) {
 
 	log.Functionf("MaybeAddResolveConfig for %s", cs.ContentID)
 	resolveConfig := types.ResolveConfig{
-		DatastoreID: cs.DatastoreID,
+		DatastoreID: cs.DatastoreIDsList[0],
```

Comment:
Let's see if the resolver code can be made to try across all the elements. First step would be to pass the list in the ResolveConfig.

Reply:
Actually, there is a design issue whether the the handleResolveConfig code should check if there are more elements in the list and try subsequent ones when there is an error, OR have the main code in volumemgr do that loop/retry on tag resolution failure.

Reply:
Good point @giggsoff . So we should park the OCI tag resultion for future work. To avoid any confusing errors it would make sense to add a check that at most the first datastore in the list can be OCI.

Reply:
There is a check, which returns an error if there is at least one OCI and the list has more than 1 elements. Can change the code and still take the same Sha resolution path if the fist element is OCI, others not. Otherwise throw an error